### PR TITLE
Story

### PR DESCRIPTION
**Pull request by nesclark**
webpage complete and ready for trainer review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,112 @@
...
+		<img src="nick.jpeg" alt="Picture of Nick" />
+		<div class="hero-text-container">
+			<div>
+			<h1>Hi, I'm Nick</h1>
+			<p>I'm not quite yet a developer, I'm still learning. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah.</p>
+			</div>
+		</div>
+	</section>
+	<section>
+		<div class="hero-text-container-mobile">
```
Comment (charliecog) [inline index.html]: repeated block of code isnt needed. can achieve your layout with just the one version

File: styles.css [hunk 2]
```diff
@@ -1,3 +1,169 @@
...
+nav img{
+	display: none;
+}
+
+.hero-text-container-mobile{
+	display:none;
+}
+
+section{
+	color: white;
```
Comment (charliecog) [inline styles.css]: not consistent colour selection type

File: styles.css [hunk 3]
```diff
@@ -1,3 +1,169 @@
 * {
 	box-sizing: border-box;
-}
\ No newline at end of file
+}
+
+body{
```
Comment (charliecog) [inline styles.css]: best practice is to have a space after your selector before the opening curly brace (please amend for all selectors)

File: styles.css [hunk 4]
```diff
@@ -1,3 +1,169 @@
...
+li{
+    list-style-type: none;
+    margin-right: 25px;
+}
+
+a{
+	text-decoration: none;
+}
+a:link{
+	color: white;
```
Comment (charliecog) [inline styles.css]: not consistent colour selection type

File: styles.css [hunk 5]
```diff
@@ -1,3 +1,169 @@
...
+}
+a:visited{
+	color: white;
```
Comment (charliecog) [inline styles.css]: not consistent colour selection type

File: index.html [hunk 6]
```diff
@@ -1 +1,112 @@
...
+			<div>
+			<h1>Hi, I'm Nick'</h1>
+			<p>I'm not quite yet a developer, I'm still learning. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah. Blah blah blah.</p>
+			</div>
+		</div>
+	</section>
+</header>
+
+<main id="portfolio">
+	<section>
```
Comment (charliecog) [inline index.html:43]: not sure what this layer of section is doing. could potentially be removed - if it has styles on it they could potentially be moved

File: styles.css [hunk 7]
```diff
@@ -1,3 +1,169 @@
...
+	margin-bottom: 40px;
+}
+
+.project-container:hover{
+	filter: brightness(1.5) saturate(1) contrast(1.2);
+}
+
+.project-description{
+	background-color: #264653;
+	width: 350px;
```
Comment (charliecog) [inline styles.css]: this is what is giving the whole article its width, it should really be done on the article itself eg: 33% if you need 3 to sit next to each other.

File: styles.css [hunk 8]
```diff
@@ -1,3 +1,169 @@
...
+	height: 250px;
```
Comment (charliecog) [inline styles.css]: you might run into issues when you have different amounts of text for each box (some more than the current space) - perhaps chaing to min-height might help allieviate?

File: styles.css [hunk 9]
```diff
@@ -1,3 +1,169 @@
...
+}
+.contact-info img{
+	width: 10%;
+}
+.contact-info img:hover{
+	width: 12%;
+}
+
+#contact section{
+	margin-left: 20%;
```
Comment (charliecog) [inline styles.css]: this is here to try and center the content but the approach isnt consistent with the approach taken on the rest of the site. might be worth a refactor to use flex on the parent and justify-content center

**Review by charliecog — COMMENTED** [review]
Really good work, its coming along great - almost there, couple little bits to tweak then you are all good. 

General comments:
- anything clickable eg: contact logos needs cursor: pointer

File: index.html (hunk 1 above)
Comment (nesclark) [reply index.html]: all done thanks

File: styles.css (hunk 3 above)
Comment (nesclark) [reply styles.css]: have changed as requested

File: styles.css (hunk 2 above)
Comment (nesclark) [reply styles.css]: have amended to hex code

File: styles.css (hunk 4 above)
Comment (nesclark) [reply styles.css]: amended to hex code

File: styles.css (hunk 5 above)
Comment (nesclark) [reply styles.css]: amended to hex code

File: styles.css (hunk 7 above)
Comment (nesclark) [reply styles.css]: you fixed this for me!

File: styles.css (hunk 8 above)
Comment (nesclark) [reply styles.css]: you fixed this for me!

File: styles.css (hunk 9 above)
Comment (nesclark) [reply styles.css]: all done (with a little help)

**Comment by nesclark** [comment]
> Really good work, its coming along great - almost there, couple little bits to tweak then you are all good.
> 
> General comments:
> 
>     * anything clickable eg: contact logos needs cursor: pointer

done :)

**Review by charliecog — APPROVED** [review]

**Pull request merged**